haystack/json-schemas/haystack-pipeline-master.schema.json

```diff
@@ -43,27 +43,12 @@
           {
             "$ref": "#/definitions/InMemoryKnowledgeGraphComponent"
           },
-          {
-            "$ref": "#/definitions/Milvus2DocumentStoreComponent"
-          },
-          {
-            "$ref": "#/definitions/OpenDistroElasticsearchDocumentStoreComponent"
-          },
-          {
-            "$ref": "#/definitions/OpenSearchDocumentStoreComponent"
-          },
-          {
-            "$ref": "#/definitions/PineconeDocumentStoreComponent"
-          },
```

Comment:
Where did these go?

Reply:
That was a small accident, should be good now